rdtools/analysis.py

```diff
@@ -434,7 +436,12 @@ def srr_soiling(self, aggregated, aggregated_insolation, **kwargs):
                                           confidence interval
             'calc_info' : Dict of detailed results (see soiling.soiling_srr() docs)
         '''
+        # suppress RdTools experimental warning
+        import warnings
+        warnings.filterwarnings("ignore")
         from rdtools import soiling
+        warnings.resetwarnings()
```

Comment:
I think libraries should avoid `warnings.resetwarnings()` because it would reset any filters that were defined outside of rdtools in addition to the one set here.  Could we use `warnings.catch_warnings` instead?  It restores the previous list of filters when the context manager exits so any user-defined filters would still be in place after this function runs. 

```python
with warnings.catch_warnings():
    warnings.simplefilter('ignore')
    from rdtools import soiling
```

Reply:
Thinking this through in a little more detail perhaps we want to continue to emit warnings when experimental submodules are used. Someone accessing those through this class should still be alerted their answers and API might change.

Reply:
![Screen Shot 2020-11-17 at 12 46 47 PM](https://user-images.githubusercontent.com/12532260/99439935-175b4500-28d3-11eb-8052-bf48415d1748.png)
Confirmed that removing the code to catch warnings, that the warnings only get emitted when the experimental functionality is accessed.